ASV44/DailyPlanner
# Refactoring

### Background


First implementation of daily planner app is full of redundant code which should be optimized, that should include rewriting instantiation and handling of UI components, managing of business logic, managing of persisted cache and dividing app into architecture layers.

### What has been done?
1. Separate main storyboard in dedicated screens storyboards
2. Change wrong implementation of screen navigation to canonical
3. Add dedicated protocols and implementation for managing cached data and data transformation
4. Adopt iOS 13
5. Increase deployment target to iOS 10.0

### How to test?
1. Build and check app UI, some of functionalities could not work because of continues refactor
